dvc/output/local.py

```diff
@@ -16,82 +15,52 @@
 
 class OutputLOCAL(OutputBase):
     REMOTE = RemoteLOCAL
+    sep = os.sep
 
-    def __init__(
-        self,
-        stage,
-        path,
-        info=None,
-        remote=None,
-        cache=True,
-        metric=False,
-        persist=False,
-        tags=None,
-    ):
-        super(OutputLOCAL, self).__init__(
-            stage,
-            path,
-            info,
-            remote=remote,
-            cache=cache,
-            metric=metric,
-            persist=persist,
-            tags=tags,
-        )
-        if remote:
-            p = os.path.join(
-                remote.prefix, urlparse(self.url).path.lstrip("/")
-            )
+    def _parse_path(self, remote, path):
+        parsed = urlparse(path)
```

Comment:
does it return a posix path on windows?

Reply:
`._parse_path()` return `PathInfo` object, which means `WindowsPathInfo` on windows and `PosixPathInfo` otherwise.

Reply:
I mean - what does urlparse return on Windows? does it convert to posix?

Reply:
It doesn't and not supposed to. When remote is used then path is `remote://remote-name/path/to/out`, so it's posix anyway.